src/lib/containers/RssFeedCards.tsx

```diff
@@ -135,25 +135,25 @@ export default class RssFeedCards extends React.Component<RssFeedCardsProps, Rss
                           href={`${this.state.rssFeed.link}/?tag=${categoryName.replace(' ', '-')}`}
                           className="SRC-no-underline-on-hover"
                           target="_blank" rel="noopener noreferrer">
-                            <div key={`${item.guid}_${categoryName}`} className="RssFeedItemCategory">{categoryName}</div>
+                            <div key={`${item.guid}_${categoryName}`} className="FeedItemCategory">{categoryName}</div>
                         </a>
                       })}
                     </div>
-                    <p className="RssFeedItemDate">
+                    <p className="FeedItemDate">
                       {moment(item['isoDate']).format('MMMM YYYY')}
                     </p>
-                    <p className="RssFeedItemTitle">{item['title']}</p>
-                    <div className="RssFeedItemDescription"
+                    <p className="FeedItemTitle">{item['title']}</p>
+                    <div className="FeedItemDescription"
                     >{item['contentSnippet'].replace(/\[...\]|\[…\]/gm, '…')}</div>
-                    <a className="RssFeedItemLink" href={item['link']} target="_blank">Continue reading</a>
+                    <a className="FeedItemLink" href={item['link']} target="_blank">Continue reading</a>
```

Comment:
Code scanning flag this line for "Potentially unsafe external link" because we use target="_blank". I think adding a rel="noopener" to the anchor tag will remove the warning but I think it is optional because we are just linking to our own Sage content anyway.